.github/workflows/build.yml

```diff
@@ -21,9 +21,9 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.1", "8.2", "8.3" ]
-                symfony: [ "^5.4", "^6.0" ]
-                sylius: [ "~1.12.0", "~1.13.0" ]
+                php: [ "8.1" ]
```

Comment:
8.2 and 8.3 are not supported?

Reply:
They are. But if it works on 8.1 it will also work on 8.2 and 8.2. 
Remember this is matrix so instead of 8 pipelines we will end up with 24... Is it worth it? Will it really test something?

Reply:
IMO not always works, Sometimes on bigger php version could be installed other version of packages which could be broken or incompatible. It's a public repository so gh actions are for free as I know. I don't block this pr because of this so it's up to you to make this changes or not :smile:

Reply:
I did it :D